### DISCUSSION TITLE
Copilot for student

### DISCUSSION THREAD

**Discussion by ekurer** [community/community#56305] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I was verfied as a student but Copilot is still not active. Can you assist?

**Comment by davecheney** [comment, marked as answer]
Please visit [/github-copilot/free_signup](https://github.com/github-copilot/free_signup) and signup.

**Comment by Sparrowehawk** [comment]
When I click on that link, all I get is the prompt to pay even though I am verified as a student

**Reply by davecheney** [reply]
@Sparrowehawk to qualify for a student plan you must have applied for the GitHub Student Developer pack which will provide you with a coupon to apply to your github account. Once the coupon is applied to your account, visit the signup link.

**Reply by SandraVoss** [reply]
how/where to i do that? i am on this side https://education.github.com/pack#offers currently and find no suitable option including copilot

**Comment by nishanprime** [comment]
same issue and it is still not solved.

**Comment by quantrung9** [comment]
same issue here :(

**Reply by quantrung9** [reply]
I finally got in 🥳 I had to wait for 2 days to receive the email notification that I joined the global campus program, and then I was able to subscribe to the CoPilot with $0 fee.

**Comment by brunoottonelli** [comment]
same issue :(

**Comment by sprynkler** [comment]
Same issue, any solutions?
